providers/facebook.go

```diff
@@ -49,14 +47,6 @@ func NewFacebookProvider(p *ProviderData) *FacebookProvider {
 	return &FacebookProvider{ProviderData: p}
 }
 
-func getFacebookHeader(accessToken string) http.Header {
-	header := make(http.Header)
-	header.Set("Accept", "application/json")
-	header.Set("x-li-format", "json")
```

Comment:
I take it this was an error in the existing codebase that you are rectifying by omitting this header moving forward?

Reply:
Indeed, this is a linked in header that somehow made it into the facebook provider